.github/workflows/test_comment.yaml

```diff
@@ -0,0 +1,16 @@
+name: test_comment
```

Comment:
Do you want to remove this and test_label.yaml?

Reply:
Yes, those can be removed, but the yaml from test_label.yaml can be used as a reference to trigger workflows based on labels.  We are planning to have some "expensive" test workflows triggered manually by users by adding a label (e.g. "run_on_device_test" label).